server/src/main/java/io/unitycatalog/server/persist/TableRepository.java

```diff
@@ -27,13 +28,38 @@ public class TableRepository {
   private final SessionFactory sessionFactory;
   private final Repositories repositories;
   private final FileOperations fileOperations;
+  private final ServerProperties serverProperties;
   private static final PagedListingHelper<TableInfoDAO> LISTING_HELPER =
       new PagedListingHelper<>(TableInfoDAO.class);
 
-  public TableRepository(Repositories repositories, SessionFactory sessionFactory) {
+  public TableRepository(
+      Repositories repositories, SessionFactory sessionFactory, ServerProperties serverProperties) {
     this.repositories = repositories;
     this.sessionFactory = sessionFactory;
     this.fileOperations = repositories.getFileOperations();
+    this.serverProperties = serverProperties;
+  }
+
+  public String getStorageLocationForTableOrStagingTable(String tableId) {
+    return TransactionManager.executeWithTransaction(
+        sessionFactory,
+        session -> {
+          LOGGER.debug("Getting storage location of table by id: {}", tableId);
```

Comment:
nit:  Please left a blank line to separate the different code blocks, which make it will be more easier to read.